pkg/cvo/updatepayload.go

```diff
@@ -246,15 +246,15 @@ func (r *payloadRetriever) fetchUpdatePayloadToDir(ctx context.Context, dir stri
 				setContainerDefaults(corev1.Container{
 					Name: "move-operator-manifests-to-temporary-directory",
 					Command: []string{
-						"mv",
+						"cp",
 						filepath.Join(payload.DefaultPayloadDir, payload.CVOManifestDir),
 						filepath.Join(tmpDir, payload.CVOManifestDir),
 					},
 				}),
 				setContainerDefaults(corev1.Container{
 					Name: "move-release-manifests-to-temporary-directory",
```

Comment:
nit: rename `move-...` here and above to `copy-...` to keep up with the `mv` -> `cp` implementation change?